occurrences/models.py

```diff
@@ -105,6 +105,17 @@ def delete_contact_info(self, now=None):
             contact_info_deleted_at=now,
         )
 
+    def with_next_occurrence(self):
+        next_occurrences = Occurrence.objects.filter(
+            p_event=OuterRef("pk"), start_time__gte=timezone.now(), cancelled=False
+        ).order_by("start_time")
+
+        return self.annotate(
+            next_occurrence_start_time=Subquery(
```

Comment:
How about `with_next_occurrence_start_time` instead of `with_next_occurrence`? That would specify exactly what is annotated, and still it's not yet overly long IMHO.